Allow to suppress prettyPrint flush sync warning (#841)
pinojs/pino: docs/pretty.md

  ```
  See the [`pino-pretty` documentation][pp] for more information on the options
  that can be passed via `prettyPrint`.
   

>   Pretty stream doesn't guarantee final log writes and corresponding warning is written to logs on 
>   first synchronous flushing.
>   To disable this warning, e.g. if you use pretty stream consciously and know that some logs can be lost,
>   you can pass `suppressFlushSyncWarning : true` to prettyPrint:

Reviewer: ```suggestion
The default prettifier write stream does not guarantee final log writes.
Correspondingly, a warning is written to logs on first synchronous flushing.
This warning may be suppressed by passing `suppressFlushSyncWarning : true` to
`prettyPrint`:
```

Author: Thanks!
